.github/ISSUE_TEMPLATE/release-check-list.md

```diff
@@ -0,0 +1,26 @@
+---
+name: New Release Check List
+about: v<TARGET_RELEASE>
+title: "v<TARGET_RELEASE Check List"
+---
+
+## v<TARGET_RELEASE>
+
+* [ ] - Get the latest ocicrypt-rs used as part of image-rs
+* [ ] - Get the image-rs, using the latest ocicrypt-rs, as part of attestastion-agent
```

Comment:
Maybe I'm reading this/understanding it incorrrectly, but I think that the attestation-agent is part of image-rs, not the other way around? This cascades to the next for items as well.